core/controllers/skill_editor_test.py

```diff
@@ -314,14 +314,16 @@ def test_editable_skill_handler_put_fails(self):
         self.put_json(
             self.url, self.put_payload, csrf_token=csrf_token,
             expected_status_int=400)
-        self.put_payload['version'] = -1
+        self.put_payload['version'] = 10
```

Comment:
Made the version some random number other than 1 to increase the tests coverage, (-1 would fail schema validation)